tests/guilds/editGuildMfaLevel.test.ts

```diff
@@ -0,0 +1,12 @@
+import { MfaLevels } from "../../mod.ts";
+import { assertEquals } from "../deps.ts";
+import { loadBot } from "../mod.ts";
+import { CACHED_COMMUNITY_GUILD_ID } from "../utils.ts";
+
+Deno.test("[guild] edit guild mfa level", () => {
+  const bot = loadBot();
+  let elevated = bot.helpers.editGuildMfaLevel(CACHED_COMMUNITY_GUILD_ID, MfaLevels.Elevated, "test");
+  assertEquals(elevated, MfaLevels.Elevated);
```

Comment:
We want it to return null so after changing it, you should fetch the the guild each time u change it here